lib/HttpSignatureStrategy.js

```diff
@@ -229,15 +229,23 @@ function getHttpsPublicKey(publicKey, callback) {
   async.auto({
     dereferencePublicKey: function(callback) {
       jsonRequest.get(publicKey, function(err, res) {
-        callback(err, res.body);
+        if(err) {
+          return callback(new BedrockError('PublicKey URL unavailable.',
```

Comment:
Style: Should drop `'PublicKey URL unavailable.'` down a line.

Note that there's another `jsonRequest` call below for getting the public key owner; if we're adding an extra error here, we should do the same below. Also, we should pass `err` as the cause to the error so our logging is more useful.